css/brand.css

```diff
@@ -41,6 +41,18 @@
   --cd-container-padding-tablet: 1.5rem; /* 24px */
   --cd-container-padding-xlarge: 2.5rem; /* 40px */
 
+  /**
+   * Breakpoints
+   *
+   * Mostly to document the old Sass vars, these values aren't used directly in
+   * CSS when needed for a media query, because custom props don't work in MQs.
```

Comment:
Whoops, I accidentally commented on the commit instead of the PR. Reiterating here: I think the original wording is correct: the _values_ are used directly in media queries, e.g. `768px` instead of `var(--cd-bp--md)`

Reply:
Ah ok, I'll revert